.travis.yml

```diff
@@ -1,5 +1,7 @@
-language: c
+language: c, go
 sudo: required
+go:
+    "1.13"
```

Comment:
This change seems not to be related.

Reply:
It's probably a merge or rebase... I don't want change myself ...

it comes probably from https://github.com/curl/curl/commit/ac34c70d7b999285aa1878548f76af38d05ffaf7